e2e/pages/app/toolbar-search/toolbar-search.testpage.component.ts

```diff
@@ -14,20 +14,37 @@ export class ToolbarSearchTestPageComponent {
     searchedFor: string = '';
     alwaysExpanded: boolean = false;
 
+    toolbarExpanded: boolean;
```

Comment:
I don't think `toolbarExpanded` is every used in the HTML

Reply:
toolbarExpanded is used below in the onToolbarSearch

Reply:
Yes, but its not used in the HTML so it isn't actually doing anything?